drivers/android/allocation.rs

```diff
@@ -19,6 +22,7 @@ pub(crate) struct Allocation<'a> {
     pub(crate) process: &'a Process,
     allocation_info: Option<AllocationInfo>,
     free_on_drop: bool,
+    pub(crate) file_list: List<Box<FileInfo>>,
```

Comment:
Most of these modules started off in another module with access to all fields. Eventually I pushed some of these out to their own modules and made most fields private; for the ones where this wasn't trivial I just made them `pub(crate)` because I had (and still do) bigger changes to worry about. I didn't want to have pure getters/setters as in Java.

For this particular change, I already have `take_file_list` (added later), so I may as well add a file_list push and make `file_list` private. Will do.

Reply:
Got ya. It's a WIP.